### ISSUE TITLE
using 3d_cascade_fullres for small clot segmentation

### ISSUE THREAD

**Issue by AugustKRZhu** [MIC-DKFZ/nnUNet#766]
Hi Fabian,

we used the nnunet to train our dataset：

-     2 channels: thin NCCT and 2rd CTA
-     labels：0,1
-     training set: 329
-     test set: 170

the nnunet automately setup two stages: 

-     stage0, 3D_lowres, 
-     stage1, 3D_fullres.

The average volume of the small clots is about 0.115ml.

The problem I encountered is that if the clot is not detected in stage0 3D_lowres, according to the idea of ​​3d_cascade_fullres, the prediction result of 3D_lowres is used as the input of stage1, then it still cannot be detected in stage1 3D_fullres, that is to say, the detection of small targets by 3D_lowres may be certain limitations. Is my understanding correct? How should it be improved? Looking forward to your reply. Thanks.

Best,
